Setup/UpgradeSchema.php

```diff
@@ -49,6 +49,35 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
                 );
         }
 
+        $swedbankPayQuoteTable = 'swedbank_pay_quotes';
```

Comment:
Adjust the Codacy ruleset or shorten the name of this variable to less than 20 characters as suggested?